spec.html

```diff
@@ -30493,24 +30492,20 @@ <h2>Syntax</h2>
           &lt;ZWJ&gt;
 
         RegExpUnicodeEscapeSequence[U] ::
-          [+U] `u` LeadSurrogate `\u` TrailSurrogate
-          [+U] `u` LeadSurrogate
-          [+U] `u` TrailSurrogate
-          [+U] `u` NonSurrogate
+          [+U] RegExpUnicodeSurrogatePair
+          [+U] [lookahead &notin; RegExpUnicodeSurrogatePair] `u` Hex4Digits
```

Comment:
`RegExpUnicodeSurrogatePair` generates a set of terminal sequences, each of length 11. This doesn't fit with the current definition of lookahead-restrictions.